packages/eslint-plugin-twenty/src/rules/no-state-useref.ts

```diff
@@ -0,0 +1,68 @@
+import { ESLintUtils } from "@typescript-eslint/utils";
+
+const createRule = ESLintUtils.RuleCreator(() => `https://docs.twenty.com`);
+
+const noStateUseRef = createRule({
+  create: (context) => {
+    return {
+      CallExpression: (node) => {
+        if (
+          node.callee.type !== "Identifier" ||
+          node.callee.name !== "useRef"
+        ) {
+          return;
+        }
+
+        if (!node.typeArguments || !node.typeArguments.params?.length) {
```

Comment:
using typeArguments instead of typeParameters which is deprecated :)

Reply:
@charlesBochet I saw this and made the necessary changes but then reverted them after I noticed the rule wasn't behaving as expected. This is because the AST generated by `@typescript-eslint/parser` uses `typeParameters` and not `typeArguments`